src/js/state/Investigation/selectors.ts

```diff
@@ -1,12 +1,23 @@
 import {State} from "../types"
+import {Finding} from "./types"
 import {last} from "../../lib/Array"
 
-export default {
-  getInvestigation(state: State) {
-    return state.investigation
-  },
+type Id = string | null
 
-  getCurrentFinding(state: State) {
-    return last(state.investigation)
+export const getInvestigation = (connId: Id, spaceId: Id) => (
```

Comment:
What do you think about a `Current.getInvestigation()` method as well, instead of having to get all these spaces and connection ids?

Reply:
I had thought of that but saw that we needed the connId and spaceId for deleting individual and groups of investigations anyway so decided to just keep current focused on the current context (defining "context" as a connection + space)